src/dtgtk/thumbtable.c

```diff
@@ -426,6 +426,36 @@ static int _thumbs_remove_unneeded(dt_thumbtable_t *table)
   return changed;
 }
 
+// change thumbnails group id
+void dt_thumbtable_group_change_representative(dt_thumbtable_t *table, const int group_id, const int new_group_id)
+{
+  if(table->mode != DT_THUMBTABLE_MODE_FILEMANAGER) return;
```

Comment:
iirc, it's also possible to "play" with groups on zoomable and filmstrip, so not sure we need this check. (same for lines above)

Reply:
On culling mode there is no way to select several images (at least I haven't found it).
On filmstrip, the G icon doesn't appear> I've also tried the shortcut but that does nothing...

Reply:
> it's also possible to "play" with groups on zoomable and filmstrip, so not sure we need this check.

I have doubt here, I use dt_ui_thumbtable(darktable.gui->ui) to identify the thumbtable. Is it valid also for zoomable and filmstrip ?

Reply:
I've removed this test. It works now for zoomable and filmstrip.